### PR TITLE
Add Dongluo to the MAINTAINERS file

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <victorvieux@gmail.com>

### CODE REVIEW

**Comment by abronan** [comment]
Hmmm I'm not sure... kidding!

LGTM :tada: :ghost:

File: MAINTAINERS [hunk 1]
```diff
@@ -42,6 +43,11 @@
 	Email = "chanwit@gmail.com"
 	GitHub = "chanwit"
 
+	[people.dongluochen]
+	Name = "Dongluo Chen"
+	Email = "dong@docker.com"
```
Comment (dongluochen) [inline MAINTAINERS]: dongluo.chen@docker.com

**Comment by dongluochen** [comment]
LGTM. Thanks Victor!

**Comment by MHBauer** [comment]
Yay!

LGTM

**Comment by moxiegirl** [comment]
LGTM to me.  Ignore nor the docs build break (thanks for checking @vieux :-D)

**Pull request merged**